com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/app/otaserver/ZclOtaUpgradeServer.java

```diff
@@ -31,6 +31,7 @@
 import com.zsmartsystems.zigbee.zcl.clusters.ZclOtaUpgradeCluster;
 import com.zsmartsystems.zigbee.zcl.clusters.otaupgrade.ImageBlockCommand;
```

Comment:
Should we call it `Image Block Request`? Both requests and responses are commands. When we call it Image Block Command it is not clear whether it is a response or a request.
Same applies to other OTA Commands that are Requests:
Image Block Request
Image Page Request
Upgrade End Request
Query Next Image Request
Query Specific File Request

Reply:
> Should we call it ImageBlockRequest

Possibly it would be better, however for historical reasons this has always been the name and I don't want to get into major refactoring.

Reply:
Ok